### DISCUSSION TITLE
Link to non-markdown/rst file in TOC

### DISCUSSION THREAD

**Discussion by jkmacc-LANL** [executablebooks/meta#477] [Q&A]
Hi, all.  I'm loving MyST, but having trouble finding documentation for how to include a link to a PDF file stored in `_static` in my TOC tree.  I like having a few important project documents available from the table of contents.  I've tried variations on the following in my `index.md`:

````
```{toctree}
:maxdepth: 2
:hidden:
:caption: Project Documents

2020 Annual Report <_static/annual_report_2020.docx>
2020 Annual Report <_static/annual_report_2020>
[2020 Annual Report](_static/annual_report_2020.docx)
![2020 Annual Report](_static/annual_report_2020.docx)
{download}`2020 Annual Report <_static/annual_report_2020.docx>`
```
````

...all of which generally result in: ` WARNING: toctree contains reference to nonexisting document '_static/annual_report_2020.docx'`

Does anyone have suggestions?  Thank you in advance!

**Comment by choldgraf** [comment]
I think of you want to include those in the table of contents, you'll have to hard code the URLs as external links. Can you try that?
